packages/browser/test/integration/test.js

```diff
@@ -617,6 +617,54 @@ for (var idx in frames) {
           );
         });
 
+        it('should capture unhandledrejection with error', function(done) {
+          var iframe = this.iframe;
+
+          iframeExecute(
+            iframe,
+            done,
+            function() {
+              setTimeout(function() {
+                return Promise.reject(new Error('test2'));
+              });
+            },
+            function(sentryData) {
+              if (debounceAssertEventCount(sentryData, 1, done)) {
+                assert.equal(sentryData[0].exception.values[0].value, 'test2');
+                assert.equal(sentryData[0].exception.values[0].type, 'Error');
+                assert.isAtLeast(sentryData[0].exception.values[0].stacktrace.frames.length, 1);
+                assert.equal(sentryData[0].exception.mechanism.handled, false);
+                assert.equal(sentryData[0].exception.mechanism.type, 'onunhandledrejection');
+                done();
+              }
+            }
+          );
+        });
+
+        it('should capture unhandledrejection as string', function(done) {
+          var iframe = this.iframe;
+
+          iframeExecute(
+            iframe,
+            done,
+            function() {
+              setTimeout(function() {
+                return Promise.reject('test');
+              });
+            },
+            function(sentryData) {
+              if (debounceAssertEventCount(sentryData, 1, done)) {
+                assert.equal(sentryData[0].exception.values[0].value, 'test');
+                assert.equal(sentryData[0].exception.values[0].type, 'Unhandled Promise Rejection');
```

Comment:
What if we'd write it in PascalCase instead? `type` is always a constructor of the error object